scripts/mod_loader/altered/misc.lua

```diff
@@ -231,3 +235,52 @@ PointList.GetLuaString = function(self)
 	return string.format("CreatePointList(%s)", save_table(extract_table(self)))
 end
 PointList.GetString = PointList.GetLuaString
+
+local spaceDamage = {
+	"bEvacuate",
+	"bHide",
+	"bHidePath",
+	"bSimpleMark",
+	"fDelay",
+	"iAcid",
+	"iDamage",
+	"iFire",
+	"iFrozen",
+	"iPawnTeam",
+	"iPush",
+	"iShield",
+	"iSmoke",
+	"iTerrain",
+	"loc",
+	"sAnimation",
+	"sImageMark",
+	"sItem",
+	"sPawn",
+	"sScript",
+	"sSound"
+}
+
+function SerializeSpaceDamage(self)
```

Comment:
"Serialization" typically refers to converting an object to a string/byte format that can be saved to a file. Here we're converting to a table, so the name doesn't quite fit. Something like `SpaceDamageToTable`  might be more descriptive

Reply:
Right, I suppose the name would have made more sense if the save_table call was inside the function.